sub-packages/bionemo-scdl/src/bionemo/scdl/index/row_feature_index.py

```diff
@@ -28,6 +28,20 @@
 __all__: Sequence[str] = ("RowFeatureIndex",)
 
 
+def are_dicts_equal(dict1, dict2):
```

Comment:
could you move the type hints to the definition line? Might also be better to re-phrase the function name to make it more explicit what it returns
```suggestion
def dicts_are_equal(dict1: dict[str, np.ndarray], dict2: dict[str, np.ndarray]) -> bool:
```